packages/patternfly-4/react-core/src/components/Popover/Popover.test.tsx

```diff
@@ -62,3 +62,20 @@ test('popover passes along values to tippy.js', () => {
   );
   expect(view).toMatchSnapshot();
 });
+
+test('popover with custom triggers', () => {
```

Comment:
this is great.  Can you also add a demo to the demo app and an integration test for this.

Reply:
Yeah sure! I added a test into its spec to make sure that setting a focus trigger will display the popover. :man_technologist: